src/DGtal/dec/PolygonalCalculus.h

```diff
@@ -435,7 +447,7 @@ class PolygonalCalculus
     if (checkCache(DIVERGENCE_,f))
       return myGlobalCache[DIVERGENCE_][f];
  
-    DenseMatrix op = D(f).transpose() * M(f);
+    DenseMatrix op = -1.0 * D(f).transpose() * M(f);
```

Comment:
Please comment in the method description that the change in sign is just a convention and that this one does not match with @cite degoes2020discrete

BTW, this convention does not match with Gradi's (positive values on diag)... It's fine for me if it is discussed

Reply:
done